diff --git a/Cargo.lock b/Cargo.lock
index 9925c0ce16..641b4a8854 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -265,9 +265,9 @@ checksum = "e2abad23fbc42b3700f2f279844dc832adb2b2eb069b2df918f455c4e18cc646"
 
 [[package]]
 name = "libc"
-version = "0.2.106"
+version = "0.2.107"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "a60553f9a9e039a333b4e9b20573b9e9b9c0bb3a11e201ccc48ef4283456d673"
+checksum = "fbe5e23404da5b4f555ef85ebed98fb4083e55a00c317800bc2a50ede9f3d219"
 
 [[package]]
 name = "libm"
diff --git a/Cargo.toml b/Cargo.toml
index 46f2cfea38..c2b79d29f2 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "artichoke"
 version = "0.1.0-pre.0"
diff --git a/artichoke-backend/Cargo.toml b/artichoke-backend/Cargo.toml
index 1cfd1e3842..e31f9f651b 100644
--- a/artichoke-backend/Cargo.toml
+++ b/artichoke-backend/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "artichoke-backend"
 version = "0.2.0"
diff --git a/artichoke-core/Cargo.toml b/artichoke-core/Cargo.toml
index 49ff35d26e..6d90d98a83 100644
--- a/artichoke-core/Cargo.toml
+++ b/artichoke-core/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "artichoke-core"
 version = "0.10.0"
diff --git a/artichoke-load-path/Cargo.toml b/artichoke-load-path/Cargo.toml
index 71d2e28ff9..ac7491dd2e 100644
--- a/artichoke-load-path/Cargo.toml
+++ b/artichoke-load-path/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "artichoke-load-path"
 version = "0.1.0"
diff --git a/mezzaluna-feature-loader/Cargo.toml b/mezzaluna-feature-loader/Cargo.toml
index 13d1db3de8..2b88567159 100644
--- a/mezzaluna-feature-loader/Cargo.toml
+++ b/mezzaluna-feature-loader/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "mezzaluna-feature-loader"
 version = "0.1.0"
diff --git a/scolapasta-hex/Cargo.toml b/scolapasta-hex/Cargo.toml
index 276a7e90b7..0d5e301ab1 100644
--- a/scolapasta-hex/Cargo.toml
+++ b/scolapasta-hex/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "scolapasta-hex"
 version = "0.1.0"
diff --git a/scolapasta-string-escape/Cargo.toml b/scolapasta-string-escape/Cargo.toml
index 90316100f3..bae83c45d0 100644
--- a/scolapasta-string-escape/Cargo.toml
+++ b/scolapasta-string-escape/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "scolapasta-string-escape"
 version = "0.2.0"
diff --git a/spinoso-array/Cargo.toml b/spinoso-array/Cargo.toml
index d8f4472de8..681f0a3d56 100644
--- a/spinoso-array/Cargo.toml
+++ b/spinoso-array/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "spinoso-array"
 version = "0.6.0"
diff --git a/spinoso-env/Cargo.toml b/spinoso-env/Cargo.toml
index eea4de93a7..a727d25824 100644
--- a/spinoso-env/Cargo.toml
+++ b/spinoso-env/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "spinoso-env"
 version = "0.1.0"
diff --git a/spinoso-exception/Cargo.toml b/spinoso-exception/Cargo.toml
index 89426add18..024a62b433 100644
--- a/spinoso-exception/Cargo.toml
+++ b/spinoso-exception/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "spinoso-exception"
 version = "0.1.0"
diff --git a/spinoso-math/Cargo.toml b/spinoso-math/Cargo.toml
index a946741f78..52075def23 100644
--- a/spinoso-math/Cargo.toml
+++ b/spinoso-math/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "spinoso-math"
 version = "0.2.0"
diff --git a/spinoso-random/Cargo.toml b/spinoso-random/Cargo.toml
index 2d1f3bd3ab..48eaaed545 100644
--- a/spinoso-random/Cargo.toml
+++ b/spinoso-random/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "spinoso-random"
 version = "0.1.0"
diff --git a/spinoso-regexp/Cargo.toml b/spinoso-regexp/Cargo.toml
index 8f3724d424..79d13b8416 100644
--- a/spinoso-regexp/Cargo.toml
+++ b/spinoso-regexp/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "spinoso-regexp"
 version = "0.2.0"
diff --git a/spinoso-securerandom/Cargo.toml b/spinoso-securerandom/Cargo.toml
index cd518de416..05ecbd42ae 100644
--- a/spinoso-securerandom/Cargo.toml
+++ b/spinoso-securerandom/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "spinoso-securerandom"
 version = "0.1.0"
diff --git a/spinoso-string/Cargo.toml b/spinoso-string/Cargo.toml
index 1ca95da521..9134f47fc4 100644
--- a/spinoso-string/Cargo.toml
+++ b/spinoso-string/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "spinoso-string"
 version = "0.2.0"
diff --git a/spinoso-symbol/Cargo.toml b/spinoso-symbol/Cargo.toml
index ed581cbd70..cd99bd0e58 100644
--- a/spinoso-symbol/Cargo.toml
+++ b/spinoso-symbol/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "spinoso-symbol"
 version = "0.1.0"
diff --git a/spinoso-time/Cargo.toml b/spinoso-time/Cargo.toml
index 2694f7b5c1..a5fe645ce4 100644
--- a/spinoso-time/Cargo.toml
+++ b/spinoso-time/Cargo.toml
@@ -1,3 +1,5 @@
+cargo-features = ["edition2021"]
+
 [package]
 name = "spinoso-time"
 version = "0.2.0"
